tailwind.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
       borderRadius: {
         10: '2.5rem',
       },
+      width: {
+        148: '37rem',
+      },
```

Comment:
Also let's keep tailwind config as minimum as possible.
we can use custom classes e.g. w-[37rem]